lib/analysis/modules/tests/ies_iteration.cpp

```diff
@@ -265,6 +265,7 @@ matrix_type * swap_matrix(matrix_type * old_matrix, matrix_type * new_matrix) {
 
 void test_deactivate(const char * testdata_file) {
```

Comment:
I would suggest a more descriptive test function name, e.g. something like `test_observation_reactivating...` Also in `es_testdata.cpp` there is another `test_deactivate` function.

Reply:
I have changed the function name as suggested.

I am testing reactivation as well. It is a bit tricky since there is no reactivation function. What I do is to start with identical copies testdata and testdata2.  In the first iteration, I remove an observation in testdata2 and compute the update. In the subsequent iterations, I use testdata which includes the initially removed datapoint. I have commented further on this in the test function.